app/models/ia_work.rb

```diff
@@ -165,7 +165,7 @@ def ingest_work(id)
 
     self.save!
     # now fetch the scandata.xml file and parse it
-    scandata_url = "http://#{server}#{dir}/#{URI.encode(scandata_file)}" # will not work on new format: we cannot assume filenames are re-named with their content
+    scandata_url = "http://#{server}#{dir}/#{CGI.escape(scandata_file)}" # will not work on new format: we cannot assume filenames are re-named with their content
```

Comment:
URI.encode is deprecated, use CGI.escape instead. 

(This is also why we need to use forked version of gravatar gem, old one is not updated and is still using URI.encode)